orchestra/actions/install.py

```diff
@@ -104,7 +105,7 @@ def _prepare_tmproot(self):
             touch "${TMP_ROOT}${ORCHESTRA_ROOT}/share/info/dir"
             mkdir -p "${TMP_ROOT}${ORCHESTRA_ROOT}/libexec"
             """)
-        run_script(script, environment=self.environment)
+        run_internal_script(script, environment=self.environment)
```

Comment:
Maybe we need a `run_internal_script` method in `Action` that implicitly passes `environment=self.environment`